text/src/autogluon/text/automm/data/process_image.py

```diff
@@ -400,3 +407,13 @@ def __call__(
             per_column_name: per_column_paths[idx] for per_column_name, per_column_paths in all_image_paths.items()
         }
         return self.process_one_sample(per_sample_paths, is_training)
+
+    def __getstate__(self):
+        odict = self.__dict__.copy()  # get attribute dictionary
+        del odict["train_processor"]  # remove augmenter to support pickle
+        return odict
+
+    def __setstate__(self, state):
+        self.__dict__ = state
+        if len(state["train_transform_types"]) > 0:
+            self.train_processor = self.construct_processor(self.train_transform_types)
```

Comment:
Do we need this customized pickling? It seems that `randaug` works well without it, right?

Reply:
Pickle gives error because of some lambda function in trivial augment operation.